cmd/server/main.go

```diff
@@ -193,6 +194,7 @@ func main() { //nolint: funlen
 	httpClient := retryClient.StandardClient()
 
 	var routes []transport.Route
+	var dRoutes []transport.DynamicRoute
```

Comment:
I think you can change this to `transport.DynamicRoute` instead of using a slice, can't you?

Reply:
Yes, it is possible.
I basically copied the routes approach to be flexible with further extensions. But I'll change it to single DynamicRoute.

Reply:
@Knappek  I would keep the slice.
There was another PR #83 which needs to be rewritten after refactoring. 
But this would be another DynamicRoute.